staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3603,6 +3603,37 @@ func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, load
 	return sgList, setupSg, nil
 }
 
+// sortELBSecurityGroupList returns a list of sorted securityGroupID based on it's original order built by buildELBSecurityGroupList
+// The logic is
+//	* securityGroups specified by ServiceAnnotationLoadBalancerSecurityGroups appears first in order and
+//  * securityGroups specified by ServiceAnnotationLoadBalancerExtraSecurityGroups appears last in order
```

Comment:
Nit: Comments are slightly off. How about:
```
// sortELBSecurityGroupList returns a list of sorted securityGroupIDs based on the original order 
// from buildELBSecurityGroupList. The logic is:
//  * securityGroups specified by ServiceAnnotationLoadBalancerSecurityGroups appears first in order
//  * securityGroups specified by ServiceAnnotationLoadBalancerExtraSecurityGroups appears last in order
```

Reply:
nice one :D, fixed it :D